<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Halaman Akun</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_hapus_akun">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>640</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus Akun</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_edit_akun">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>640</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Akun</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Tambah Akun Baru</string>
   </property>
   <widget class="QLineEdit" name="input_tambah_name">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string comment="&quot;00&quot;" extracomment="&quot;00&quot;"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_tambah_akun">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Tambah Akun</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_tambah_username">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string comment="&quot;00&quot;" extracomment="&quot;00&quot;"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_tambah_password">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string comment="&quot;00&quot;" extracomment="&quot;00&quot;"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_edit_akun">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>400</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit Akun</string>
   </property>
   <widget class="QLineEdit" name="input_edit_name">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string comment="&quot;00&quot;" extracomment="&quot;00&quot;"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save_edit_akun">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Akun</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_edit_username">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string comment="&quot;00&quot;" extracomment="&quot;00&quot;"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_edit_password">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string comment="&quot;00&quot;" extracomment="&quot;00&quot;"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>input_tambah_name</tabstop>
  <tabstop>input_tambah_username</tabstop>
  <tabstop>input_tambah_password</tabstop>
  <tabstop>button_tambah_akun</tabstop>
  <tabstop>button_edit_akun</tabstop>
  <tabstop>button_hapus_akun</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
